frontend/e2e/cypress/integration/search-start.spec.js

```diff
@@ -0,0 +1,44 @@
+/// <reference types="cypress" />
+
+describe('Dashboard', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000');
+  });
+  it('Dashboard should show three panels', () => {
+    cy.get('[data-cy="dashboard"]').find('div').should('have.length', 3);
+  });
+
+  it('Dashboard should scroll horizontally on mobile screens', () => {
+    cy.viewport(335, 600);
+    // if element is not scrollable, this will fail
+    cy.get('[data-cy="dashboard"]').scrollTo('right');
+  });
+});
+
+describe('Search Input', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000');
+  });
+  it('Search input placeholder text should be visible', () => {
+    cy.get('[data-cy="search-input"]').should(
+      'have.attr',
+      'placeholder',
+      "Search for something, e.g. 'carbon taxes'"
+    );
+  });
+  it('Clear search button should not be visible until typing a search term', () => {
+    cy.get('[data-cy="search-close-button"]').should('not.exist');
+    cy.get('[data-cy="search-input"]').type('carbon taxes');
+    cy.get('[data-cy="search-clear-button"]').should('exist');
+  });
+  it('Clicking the clear search button should clear the search terms in the input box', () => {
```

Comment:
@eurolife Is it worth checking that the placeholder text reappears at this point, and the clear search button disappears?

Reply:
@kdutia  I'm not sure there is a way to check that the placeholder text is visible or not but I have added a final check to ensure the button is not visible after clearing the search.

Reply:
@kdutia merging as-is, as I don't see a way to check if placeholder text is visible or not.